hphp/runtime/ext/fb/config.cmake

```diff
@@ -0,0 +1,19 @@
+HHVM_DEFINE_EXTENSION("fb"
+  PRETTY_NAME "FB Serializer"
+  SOURCES
+    ext_fb.cpp
+  HEADERS
+    ext_fb.h
+    FBSerialize.h
+    FBSerialize-inl.h
+    VariantController.h
+  EXTENSION_LIBRARY
+    ext_fb.php
+  IDL
+    fb.idl.json
+  DEPENDS_UPON
+    ext_mysql
```

Comment:
As PR #5573 has been merged before this made it in, this dependency should be removed.